### PR TITLE
Navbar finished

### PR DESCRIPTION
**Pull request by islanddrago**
i think its ready

### CODE REVIEW

**Review by joe-niu — APPROVED** [review]

File: ebay-frontend/src/app/main-nav/main-nav.component.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+import { BreakpointObserver, Breakpoints } from '@angular/cdk/layout';
+import { Observable } from 'rxjs';
+import { map, shareReplay } from 'rxjs/operators';
+
+@Component({
+  selector: 'app-main-nav',
```
Comment (mcjcloud) [inline ebay-frontend/src/app/main-nav/main-nav.component.ts:7]: rename to `navbar`

File: ebay-frontend/src/app/app.component.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<navbar></navbar>
+<app-main-nav></app-main-nav>
```
Comment (mcjcloud) [inline ebay-frontend/src/app/app.component.html:1]: `navbar`

File: ebay-frontend/src/styles.scss [hunk 3]
```diff
@@ -1,3 +1,39 @@
 /* You can add global styles to this file, and also import other style files */
 @import '@angular/material/prebuilt-themes/deeppurple-amber.css';
 
+.center {
+    display: block;
```
Comment (mcjcloud) [inline ebay-frontend/src/styles.scss:5]:
```
display: flex;
justify-content: center;
```

File: ebay-frontend/src/app/main-nav/main-nav.component.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
+.sidenav-container {
+  height: 100%;
+}
+
+.sidenav {
+  width: 200px;
+}
+
+.sidenav .mat-toolbar {
```
Comment (mcjcloud) [inline ebay-frontend/src/app/main-nav/main-nav.component.scss:9]: Won't need this section

**Pull request merged**